src/universal/modules/summary/components/MeetingSummary/MeetingSummary.js

```diff
@@ -22,10 +24,12 @@ const MeetingSummary = (props) => {
     styles,
     teamId,
     teamMembers,
-    teamName
+    teamName,
+    title
   } = props;
   return (
     <MeetingMain>
+      <Helmet title={title} {...head} />
```

Comment:
why do we need to to pass in the meta data? Aren't those already there from the html template via SSR?